MoltenVKPackaging.xcodeproj/project.pbxproj

```diff
@@ -278,6 +279,7 @@
 				A9C70F45221B04C800FBA31A /* create_dylib_ios.sh */,
 				A9C70F41221B04C800FBA31A /* create_dylib_macos.sh */,
 				A9FC5F64249D3778003CB086 /* create_dylib_tvos.sh */,
+				279BD8642B7FAE0A0021CE70 /* create_dylib_xros.sh */,
```

Comment:
Where is `create_dylib_xros.sh` itself?

Reply:
it was already there, just not added as a reference to the Xcode project

Reply:
Ah. Can you drag it into Xcode under Project Navigator->Scripts, please? That must have got overlooked when the script was created.